src/schema/nmdc.yaml

```diff
@@ -152,7 +152,7 @@ classes:
     slot_usage:
       id:
```

Comment:
I would like to move away from asserting `id` patterns in `slot_usage`s because it implies that the range is a string, not a class

Reply:
I will deal with this in the LinkML repo